tracer/tracer.go

```diff
@@ -23,11 +26,16 @@ var (
 	GlobalTracer Tracer
 )
 
+// MaxLabelSize is the maximum allowed total labels size
+const MaxLabelSize = 10 * 1024
```

Comment:
how is this value determined? (is it like other insts?)

Reply:
yep